web/themes/custom/jcc_components/templates/paragraph/paragraph--hero.html.twig

```diff
@@ -40,12 +40,12 @@
 #}
 
 {% set parent_type = paragraph.getParentEntity().getType() %}
-{% set variant = paragraph.field_variant.value|json_decode %}
+{% set no_aside = content.field_subheading.0 is empty and content.field_aside.0 is empty and content.field_link.0 is empty %}
```

Comment:
Also, let's add a comment above this line, like:
`{# Force layout to full width if "aside" fields are empty. #}`